onnxscript/function_libs/torch_aten/ops/core.py

```diff
@@ -1509,11 +1511,28 @@ def aten_embedding_sparse_backward(
     raise NotImplementedError()
 
 
-def aten_empty_like(self: TensorType, memory_format: Optional[str] = None) -> TensorType:
+"""
+@torch_op("aten::empty")
+def aten_empty(size: INT64, dtype : int = 1) -> TensorType:
+    # empty(SymInt[] size, *, ScalarType? dtype=None, Layout? layout=None, Device? device=None, bool? pin_memory=None, MemoryFormat? memory_format=None) -> Tensor
+
+    # using RandomUniform value to simulate np.empty()
+
+    result = op.RandomUniform(dtype, high=1e30, low=-1e30, shape=size)
+    return result
+"""
+
+"""
+@torch_op("aten::empty_like")
+def aten_empty_like(self, dtype: int = 7):
     # empty_like(Tensor self, *, ScalarType? dtype=None, Layout? layout=None, Device? device=None, bool? pin_memory=None, MemoryFormat? memory_format=None) -> Tensor
 
-    raise NotImplementedError()
+    # using RandomUniform value to simulate np.empty()
 
+    input_shape = op.Shape(self)
+    result = op.RandomUniform(dtype, high=1e30, low=-1e30, shape=input_shape)
```

Comment:
Shape is an attribute so we could not use a result from op.Shape() to set it. Probably op.RandomUniformLike() is more suitable.

Reply:
use Zeros() instead